daml_dit_ddit/subcommand_build.py

```diff
@@ -97,13 +98,30 @@ def walk_and_do(fn, src_dir):
         bytecode_compile=True,
         deterministic_timestamp=True)
 
+DAML_YAML_NAME = 'daml.yaml'
+
+def daml_yaml_version():
+    with open('daml.yaml', "r") as f:
```

Comment:
Should this be `DAML_YAML_NAME` in keeping with the refactor to a constant?

Reply:
Yup. Good catch.

Reply:
Resovled.